templates/test/by_id.html

```diff
@@ -67,7 +67,34 @@
                         </tbody>
                     </table>
                 {% else %}
-                    <p>This test is still in the queue! Please check again later.</p>
+                      <p>This test is still in the queue! Please check again later.Wait {{time}} minutes</p>
```

Comment:
i have calculated time by taking the last active test and equating 
time=(test['end']-test['start']).totalMintutes
as the progress (start and end time) of tests in queue is empty.